packages/ses/src/compartment-shim.js

```diff
@@ -27,9 +27,12 @@ import { getDeferredExports } from './module-proxy.js';
 import { assert } from './error/assert.js';
 import { compartmentEvaluate } from './compartment-evaluate.js';
 import { makeSafeEvaluator } from './make-safe-evaluator.js';
+import {makeHardener} from './make-hardener.js';
 
 const { quote: q } = assert;
 
+const safeHarden = makeHardener();
```

Comment:
There should be a tamed harden laying around or available to be threaded through to here from `lockdown.js` (tho be aware it moves in #1705)